studymanager-services/src/main/java/io/redlink/more/studymanager/repository/InterventionRepository.java

```diff
@@ -85,10 +87,14 @@ public List<Intervention> listInterventions(Long studyId) {
         return template.query(LIST_INTERVENTIONS, getInterventionRowMapper(), studyId);
     }
 
-    public List<Intervention> listInterventionsForGroup(Long studyId, Integer groupId) {
+    public List<Intervention> listInterventionsForGroup(Long studyId, Integer groupId){
+        return listInterventionsForGroup(studyId, groupId, List.of());
```

Comment:
```suggestion
        return listInterventionsForGroup(studyId, groupId, Collections.emptyList());
```
Benötigt weniger Speicher

Reply:
Habe im Code nachgesehen ... in beiden Fällen gibt es eine interen Implementierung von List und eine static Instanz der selben welche für jeden Aufruf verwendet wird. Es wird also in beiden Fällen nur eine 64bit Referenz im Heap abgelegt ;)